### PR TITLE
Namespaces and catalog apps 

### PR DESCRIPTION
**Pull request by westlywright**

### CODE REVIEW

File: lib/shared/app/utils/parse-version.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 // https://github.com/ef4/ember-browserify#the-workaround
 import Semver from 'npm:semver'; // eslint-disable-line no-unused-vars
-export { default } from 'shared/utils/parse-version';
+export { satisfies, parse, comparePart, compare, minorVersion } from 'shared/utils/parse-version';
```
Comment (vincent99) [inline lib/shared/app/utils/parse-version.js]: If `lib/shared/addon/components/new-catalog/component.js` is updtated to import from `shared/utils/parse-version` can't this file go away entirely?

File: lib/shared/addon/components/form-key-value/component.js [hunk 2]
```diff
@@ -122,8 +122,14 @@ export default Component.extend({
         return;
       }
 
-      var lines = str.split(/\r?\n/);
+      // if we paste an object with trailing commas the commas are not stripped
```
Comment (vincent99) [inline lib/shared/addon/components/form-key-value/component.js]: Key-value fields are arbitrary values and might a password that ends in `,`

Comment (westlywright) [reply lib/shared/addon/components/form-key-value/component.js]: okay cool, i wasn't sure about this, i'll remove it

File: lib/shared/app/utils/parse-version.js (hunk 1 above)
Comment (westlywright) [reply lib/shared/app/utils/parse-version.js]: yes it could but any engines that may need to consume the api wouldn't be able to find it.

**Pull request closed without merging**